components/script/dom/browsingcontext.rs

```diff
@@ -168,8 +156,7 @@ impl BrowsingContext {
     }
 
     pub fn clear_session_history(&self) {
-        self.active_index.set(0);
-        self.history.borrow_mut().clear();
+        self.active_document.set(None)
```

Comment:
Is this fine? This is only called when `shut_down_layout` is called, I guess we don't need to do any constellation cleanup?